### ISSUE TITLE
Minor version bump

### ISSUE THREAD

**Issue by jaroeichler** [sahasatvik/typst-theorems#43]
Can you please make a minor release to update the package in typst/packages to make it work with typst 12?

**Comment by sebaseb98** [comment]
Concretely I get the following warnings when compiling my document under typst 0.12

```
warning: `state.display` is deprecated
    ┌─ @preview/ctheorems:1.1.2/lib.typ:167:2
    │
167 │   thm-qed-done.display()
    │   ^^^^^^^^^^^^^^^^^^^^^^
    │
    = hint: use `state.get` in a `context` expression instead
```

```
warning: `locate` with callback function is deprecated
   ┌─ @preview/ctheorems:1.1.2/lib.typ:37:15
   │  
37 │         result = locate(loc => {
   │ ╭────────────────^
38 │ │         return thmcounters.update(thmpair => {
39 │ │           let counters = thmpair.at("counters")
40 │ │           // Manually update heading counter
   · │
76 │ │         })
77 │ │       })
   │ ╰────────^
   │  
   = hint: use a `context` expression instead
```

```
warning: `state.display` is deprecated
   ┌─ @preview/ctheorems:1.1.2/lib.typ:79:15
   │  
79 │         number = thmcounters.display(x => {
   │ ╭────────────────^
80 │ │         return global_numbering(numbering, ..x.at("latest"))
81 │ │       })
   │ ╰────────^
   │  
   = hint: use `state.get` in a `context` expression instead

```

```
warning: `locate` with callback function is deprecated
    ┌─ @preview/ctheorems:1.1.2/lib.typ:203:2
    │  
203 │ ╭   locate(loc => {
204 │ │     if thm-qed-done.at(loc) == false {
205 │ │       h(1fr)
206 │ │       thm-qed-show
207 │ │     }
208 │ │   })
    │ ╰────^
    │  
    = hint: use a `context` expression instead
```

```
warning: calling `query` with a location is deprecated
    ┌─ @preview/ctheorems:1.1.2/lib.typ:240:15
    │
240 │     let thms = query(selector(<meta:thmenvcounter>).after(loc), loc)
    │                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    │
    = hint: try removing the location argument
```

A fix is highly appreciated! Thank you :)

**Comment by balltar** [comment]
@sahasatvik merged the changes in #39. But I think they still need to do the version bump.

**Comment by sahasatvik** [comment]
The version has been bumped to v1.1.3 today, and is available [officially](https://typst.app/universe/package/ctheorems).
